src/drivers/px4fmu/fmu.cpp

```diff
@@ -838,8 +877,10 @@ PX4FMU::publish_pwm_outputs(uint16_t *values, size_t numvalues)
 void
 PX4FMU::work_start()
 {
+#ifndef PX4FMU_TASK
```

Comment:
@julianoes - I am in the midst of testing changes. I will be pushing changes soon where the fmu has a command line option [task] so the modality will be runtime.